package.json

```diff
@@ -57,15 +57,15 @@
     "start": "webpack-dev-server --config webpack.dev.config.js",
     "start-dev-profiling": "cross-env NODE_ENV=dev-profiling API_URL=https://daostack-alchemy.herokuapp.com S3_BUCKET=daostack-alchemy-staging webpack-dev-server --config webpack.dev.config.js",
     "start-docker": "webpack-dev-server --config webpack.docker.config.js",
-    "start-prod": "cross-env NODE_ENV=production webpack-dev-server --config webpack.prod.config.js",
+    "start-prod": "cross-env NODE_ENV=production webpack-dev-server --config webpack.dev.config.js",
```

Comment:
What is the intention here?

Reply:
The intention here is: start with "production settings" (i.e. connect to mainnet and prod subgraph) but use the  `dev` webpack (so we still have source maps and can use the devtools debugger)

it sort  of slipped into this PR, bc I needed to check if the fix works in the current production settings, but I think this is the most useful setting here.

Reply:
I imagine this will significantly increase the bundle size.  Is that acceptable?

Reply:
this command is only meant for devs; it is not actually used in production